src/embedded_runner.rs

```diff
@@ -197,7 +206,7 @@ async fn install_or_passthrough(
     }
 
     emit(&event_channel, StateSignal::IsReady).await;
-    println!("APP_WS_PORT: {}", using_app_ws_port);
+    println!("     APP_WS_PORT: {}", using_app_ws_port);
```

Comment:
? just wondering

Reply:
Just aligning the output with the next println